responsibleai/responsibleai/_internal/constants.py

```diff
@@ -83,14 +96,38 @@ class SerializationAttributes:
 
     # File structure
     RESULTS_DIRECTORY = 'results'
+    PREDICTIONS_DIRECTORY = 'predictions'
+    DATA_DIRECTORY = 'data'
+    DASHBOARD_SCHEMAS = 'dashboard_schemas'
 
     # Metadata keys
     ID_KEY = 'id'
     VERSION_KEY = 'version'
 
-    # Metadata filnames
+    # Metadata filenames
     ID_FILENAME = 'id.json'
     VERSION_FILENAME = 'version.json'
+    META_JSON = 'meta.json'
+    RAI_VERSION_JSON = 'rai_version.json'
 
     # Dashboard filenames
     DASHBOARD_FILENAME = 'dashboard.json'
+
+    # Model filenames
+    MODEL_PKL = 'model.pkl'
+
+    # Prediction filenames
+    PREDICT_JSON = "predict.json"
+    LARGE_PREDICT_JSON = "large_predict.json"
+    PREDICT_PROBA_JSON = "predict_proba.json"
+    LARGE_PREDICT_PROBA_JSON = "large_predict_proba.json"
+
+    # Data filenames
+    LARGE_TEST_JSON = "large_test.json"
+
+
+class FileFormats:
+    """Constants relating to file formats."""
+    JSON = '.json'
```

Comment:
this seems like something re-used so often it should live in the most common place possible, raiutils

Reply:
Agreed! Will add a separate PR